OrchardCore/Reset-OrchardCoreApp/Reset-OrchardCoreApp.psm1

```diff
@@ -139,6 +142,15 @@ function Reset-OrchardCoreApp
         $SetupDatabaseConnectionString = ""
         if ($PSCmdlet.ParameterSetName -eq "ServerDB")
         {
+            if ($SuffixDatabaseNameWithFolderName.IsPresent)
+            {
+                $currentPath = (Get-Location).Path
+                $solutionFolder = Split-Path (Split-Path (Split-Path $currentPath -Parent) -Parent) -Leaf
```

Comment:
Nem tetszik, hogy a könyvár struktúra így be van hard codeolva. Lehetne helyette felléptetni, pl.:
```powershell
$solutionPath = (Get-Location).Path
while (-not (Test-Path (Join-Path $solutionPath "*.sln")))
{
    $solutionPath = Split-Path $solutionPath -Parent;
}
$solutionFolder = Split-Path $solutionPath -Leaf
```

Reply:
Na igen, itt lusta voltam, mondván, hogy 95%-ban ezt mi fogjuk használni, a maradék 5 pedig vagy használjon ugyanekkora mélységet, vagy küldjön PR-t :D. De jogos, és köszi a kódot, javítottam.